hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -393,7 +399,6 @@ def do_draw(self, data):
         return result
 
 
-@check_function
```

Comment:
I don't know _why_ this started failing, but also don't think it really needs to be a check function... so it's not.